VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -221,6 +221,12 @@ def _parse_config(self, ctxt):
                     self.error('JSON exception decoding ' + cleartxt)
                 handlerSettings['protectedSettings'] = jctxt
                 self.log('Config decoded correctly.')
+                # cleaning/removing the temp files created
+                try:
+                    if os.path.isfile(f.name):
+                        os.remove(f.name)
```

Comment:
are we sure this file is not referenced anywhere else during the execution cycle of the code.

Reply:
yes